src/app/components/ProposalInfo/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+import styled from 'styled-components';
+
+import { theme, Typography } from 'app/theme';
+import { ReactComponent as CalendarIcon } from 'assets/icons/calendar.svg';
+import { ReactComponent as PieChartIcon } from 'assets/icons/pie-chart.svg';
+import { ReactComponent as UserIcon } from 'assets/icons/users.svg';
+import { normalizeContent } from 'utils';
+
+dayjs.extend(duration);
+
+export default function ProposalInfo(props) {
+  const ProposalWrapper = styled.div`
```

Comment:
The styled components need to be out of scope.
The previously written code was good.